bower.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "purescript-line-wrapping",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "output"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/slamdata/purescript-line-wrapping.git"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "purescript-prelude": "^2.0.0",
+    "purescript-console": "^2.0.0",
+    "purescript-arrays": "^3.2.1",
+    "purescript-strings": "2.1.0",
+    "purescript-integers": "2.1.1"
+  },
+  "devDependencies": {
+    "purescript-psci-support": "^2.0.0",
```

Comment:
Ok, this is the very last change request! We don't usually include `psci-support` in the `devDependencies`.